tests/unit/lib/test_datachain.py

```diff
@@ -786,6 +859,20 @@ def test_parse_tabular(tmp_dir, test_session):
     assert df1.equals(df)
 
 
+@skip_if_not_sqlite
+def test_parse_tabular_in_memory(tmp_dir):
```

Comment:
[Q] Should all the different `from_` methods be tested individually for `in_memory`?

Reply:
I tested all the methods where a Session can be specified directly, which includes `from_storage`, `datasets`, `from_values`, `from_pandas`, and `from_records`. I also added `in_memory` tests for `from_csv` and `from_parquet`. The other methods all appear to either not allow a session to be specified (which is where `in_memory` is implemented), or pass the `session` and `in_memory` args through a generic `**kwargs` argument into another method that is already tested, such as `from_json` and `from_jsonl` pass to `from_storage` for example.